src/results/electric_tariff.jl

```diff
@@ -14,7 +14,38 @@
 - `year_one_export_benefit_before_tax` export credits over the first year, before considering tax benefits. A positive value indicates a benefit. 
 - `lifecycle_coincident_peak_cost_after_tax` lifecycle coincident peak charge in present value, after tax
 - `year_one_coincident_peak_cost_before_tax` coincident peak charge over the first year
+
+Outputs related to electric tariff (year-one rates and costs; not escalated):
+- `monthly_fixed_cost` the fixed monthly cost of electricity for each month per chosen electric tariff
```

Comment:
is this a rate or the total fixed cost? If total, should it belong in the section below?